_posts/2025-03-31-zscore-hybrid-search.md

```diff
@@ -0,0 +1,215 @@
+---
+layout: post
+title:  Introducing the z-score normalization technique for hybrid search
+authors:
+  - kazabdu
+  - gaievski
+  - minalsha
+  - kolchfa
+date: 2025-03-22
+has_science_table: true
+categories:
+  - technical-posts
+meta_keywords: z-score normalization, OpenSearch 3.0-beta1, neural search plugin, hybrid search, relevance ranking, search normalization, k-nn search, L2 normalization, how reciprocal rank fusion works
+meta_description: Learn about z-score normalization in OpenSearch 3.0-beta1. Discover how this new approach to hybrid search merges results from multiple query sources for improved relevance.
```

Comment:
Please update the mta wit the following: 

_meta_keywords:_ z-score normalization, hybrid search, data normalization, OpenSearch search latency, search relevance, hybrid search score normalization

_meta_description:_ Explore z-score normalization in OpenSearch 3.0-beta1 for hybrid search. Learn how this technique compares to min-max normalization through benchmarks on search relevance and performance.

Reply:
@pajuric thanks for the review. Pushed the change